README.md

```diff
@@ -141,27 +141,26 @@ more environments.
 
 To build using Ninja, run:
 
-    cd swift
-    utils/build-script --release-debuginfo
+    swift/utils/build-script --release-debuginfo
```

Comment:
Sorry to ask for a bit more. I would go even a bit further and add a ./ at the beginning. That way someone can copy/paste into their terminal.

Reply:
Actually @gottesmm I tried it and on Linux (Ubuntu 14) it worked for me to just use `swift/utils/build-script --release-debuginfo` without even needing to use the `./` in front of it. I guess the file was already an executable.

But sure, I'll make another PR to add the `./` in front of it for consistency. I think that's a great idea.